bld/CLMBuildNamelist.pm

```diff
@@ -2523,6 +2524,11 @@ sub setup_logic_surface_dataset {
   if ($flanduse_timeseries ne "null" && &value_is_true($nl_flags->{'use_cndv'}) ) {
      $log->fatal_error( "dynamic PFT's (setting flanduse_timeseries) are incompatible with dynamic vegetation (use_cndv=.true)." );
   }
+  # Turn test option off for NEON until after XML is interpreted
```

Comment:
In meeting with @slevis-lmwg we realized there should be some comments around this tricky logic here and about why NEON is handled differently.

In the long run the add_defaults method should just always expand XML variables for ALL cases before checking for file existence. That way the logic doesn't have to be specific for NEON (and the same thing will also apply to PLUMBER2). That also means that we'll be able to add appropriate XML variables to filenames anytime they might be helpful.

Reply:
Resolving for now.